### ISSUE TITLE
How to check convergence of model?

### ISSUE THREAD

**Issue by julietmorris** [AdrianHordyk/LBSPR#18]
Hi Adrian, 
I was wondering how do you check whether the model has converged or not? Some other models have that build in and I was wondering if you had any advice or ideas to check that. When running the model my output numbers are pretty strange so I was wondering whether that might be something that I need to address and adjust for. 

Thanks in advance,
Juliet

**Comment by AdrianHordyk** [comment]
No, the package doesn't have anything so sophisticated unfortunately.  

You could modify the internal function `LBSPRfit` to return the full object from the call the `optim` which may have some useful information re convergence.

If you have any more data than a year or two of length composition data I recommend using the Rapid Conditioning Model (`RCM`) from the `SAMtool` package instead of the equilibrium LBSPR approach.

https://samtool.openmse.com/reference/RCM.html

https://openmse.com/tutorial-rcm/

**Comment by julietmorris** [comment]
Thank you so much! I will try out those options.
